Example/Tests/Tests.m

```diff
@@ -0,0 +1,9 @@
+//
+//  RaisinToastTests.m
+//  RaisinToastTests
```

Comment:
Are there actually tests for this repo? If not, then we should remove them and any reference to them